src/content/pages/partnerships.mdx

```diff
@@ -0,0 +1,116 @@
+---
+seo:
+  title: Partner with Astro
+  description: Want to partner with Astro?
+---
+
+import StatsFeature from '~/components/StatsFeature.astro'
+import Note from '~/components/PartnershipNote.astro'
+
+![](../../assets/partnerships-page-1.webp)
+
+**Let's improve the web, together!**
+
+## ❤ Sponsorships and Official Partners
+
+The easiest way to partner with Astro is to become a sponsor! Your financial contributions support the open-source project's ongoing maintenance and developent.
+
+We recognize some levels of sponsorship with a logo and/or link on Astro sites such as our homepage, GitHub README, and docs website.
+
+<Note color="purple">
+  💸 **To sponsor:** Visit [Open Collective](https://opencollective.com/astrodotbuild) or [GitHub Sponsors](https://github.com/sponsors/withastro) and choose from a list of contribution tiers. Thank you for supporting Astro!
+</Note>
+
+If you don't see your desired level of sponsorship listed, please contact us at `partner@astro.build`!
+
+We are always happy to work directly with larger sponsors ($2000+ per month) to create a custom sponsorship package. There are also limited options for being an exclusive "official" partner for a particular category (e.g. Official CMS Partner).
+
+## 📊 About Astro
+
+{/* The icons from the original are a nice touch! Can we use them here? e.g. trophy for #1 in retention */}
+
+<StatsFeature stats={[
+  { amount: "⬇ 240k", text: "monthly downloads" },
+  { amount: "🌐 79k", text: "X/Twitter Followers" },
+  { amount: "⭐ 48k", text: "GitHub Stars" },
+  { amount: "🏆 #1", text: "in developer retention" },
+  { amount: "❤ #1", text: "in developer interest" },
+  { amount: "📈 3x", text: "growth in usage" },
```

Comment:
Growth since when? Relative to what?

Reply:
Yeah, there was nothing on the original, but we could be linking (or referencing e.g. State of JS, npm downloads etc.)

Reply:
Added sources!